specification/servicefabric/resource-manager/Microsoft.ServiceFabric/2017-07-01-preview/servicefabric.json

```diff
@@ -3130,25 +3130,23 @@
       "in": "query",
       "description": "The version of the API. This is a required parameter and it's value must be \"2016-09-01\".",
       "required": true,
-      "type": "string",
-      "enum": [
-        "2016-09-01"
-      ],
-      "default": "2016-09-01"
+      "type": "string"

```

Comment:
@samedder - Can you please take a look at changes I made in the service fabric resource-manager spec. 
- Some of the global parameters missed the `"x-ms-parameter-location": "method"` extension.
- The enum array is not required in the first place and it had an incorrect value as well
-

Reply:
@chingchenmsft needs to sign off on this. I believe this may break our clients so he should verify.

Reply:
@juhacket - I believe this changes were merged in the PR https://github.com/Azure/azure-rest-api-specs/pull/1717. Can you please take a look?

Reply:
Can you fix the description as well? **"This is a required parameter and it's value must be \"2016-09-01\"."**